cmd/aro/update_ocp_versions.go

```diff
@@ -0,0 +1,176 @@
+package main
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+	"github.com/Azure/ARO-RP/pkg/database"
+	"github.com/Azure/ARO-RP/pkg/env"
+	"github.com/Azure/ARO-RP/pkg/metrics/statsd"
+	"github.com/Azure/ARO-RP/pkg/util/encryption"
+	"github.com/Azure/ARO-RP/pkg/util/keyvault"
+	"github.com/Azure/ARO-RP/pkg/util/version"
+)
+
+func getLatestOCPVersions(ctx context.Context, log *logrus.Entry) ([]api.OpenShiftVersion, error) {
+	env, err := env.NewCoreForCI(ctx, log)
+	if err != nil {
+		return nil, err
+	}
+	dstAcr := os.Getenv("DST_ACR_NAME")
+	acrDomainSuffix := "." + env.Environment().ContainerRegistryDNSSuffix
+
+	dstRepo := dstAcr + acrDomainSuffix
+	var (
+		OpenshiftVersions = []api.OpenShiftVersion{
+			{
+				Version:           version.InstallStream.Version.String(),
+				OpenShiftPullspec: version.InstallStream.PullSpec,
+				InstallerPullspec: dstRepo + "/aro-installer:release-4.10",
+				Enabled:           true,
+			},
+		}
+	)
+	return OpenshiftVersions, nil
+}
+
+func getVersionsDatabase(ctx context.Context, log *logrus.Entry) (database.OpenShiftVersions, error) {
+	_env, err := env.NewCore(ctx, log)
+	if err != nil {
+		return nil, err
+	}
+
+	for _, key := range []string{
+		"DST_ACR_NAME",
+	} {
+		if _, found := os.LookupEnv(key); !found {
+			return nil, fmt.Errorf("environment variable %q unset", key)
+		}
+	}
+
+	if !_env.IsLocalDevelopmentMode() {
+		for _, key := range []string{
+			"MDM_ACCOUNT",
+			"MDM_NAMESPACE",
+		} {
+			if _, found := os.LookupEnv(key); !found {
+				return nil, fmt.Errorf("environment variable %q unset", key)
+			}
+		}
+	}
+
+	msiAuthorizer, err := _env.NewMSIAuthorizer(env.MSIContextRP, _env.Environment().ResourceManagerEndpoint)
+	if err != nil {
+		return nil, err
+	}
+
+	msiKVAuthorizer, err := _env.NewMSIAuthorizer(env.MSIContextRP, _env.Environment().ResourceIdentifiers.KeyVault)
```

Comment:
nit: Could rename the var to be more obvious. Not blocking.

Reply:
This is duplicated from the other commands, so I think it will need to be fixed everywhere, but this is currently blocking a lot of work that needs to be done so I think it might be best if we forgo this for now